src/main/java/org/spongepowered/api/world/storage/WorldProperties.java

```diff
@@ -105,6 +106,13 @@
     String getWorldName();
 
     /**
+     * Gets the directory the files for this world are located.
+     *
+     * @return The directory the files are located in
+     */
+    File getWorldDirectory();
+
+    /**
```

Comment:
what if someone comes up with an elaborate way to store worlds in a database or something that would be very inefficient, but not file-storage?